Flowchart/Views/Home/About.cshtml

```diff
@@ -1,10 +1,20 @@
 ﻿@{
     ViewData["Title"] = "About";
 }
+
 <h2>@ViewData["Title"].</h2>
 <h3>@ViewData["Message"]</h3>
 
 
-    <p>This software is designed to provide a simple and easy way to create flowcharts for simple tasks such as mapping out how to use github similar to what <a href="http://smessina.com/gitflow/#/">one individual created</a>!</p>
+
+    <p>This software is designed to provide a simple and easy way to create flowcharts for various purposes.
+    For a list of possible problems that may arise please look below</p>
+    <dl>
```

Comment:
Add a header here instead of just text to the end of a paragraph. Look into what viewdata and viewbags are and how to manipulate them inside controllers. This needs to be modified since it doesn't look good with how it is now.